networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -80,21 +83,45 @@ def __init__(self, region, args, verbose=False, verify_ssl=False):
             self.netbox.http_session = requests.Session()
             self.netbox.http_session.verify = False
 
-    def _ensure_single_item(self, item_set: Iterable[Any], entity: str, identifier: str, attribute: str) -> Any:
+    @classmethod
+    def _ignore_filter(cls, items: Iterable[NbRecord]) -> Generator[NbRecord, None, None]:
+        for item in items:
+            if any((x.slug in cls.ignore_tags for x in getattr(item, 'tags', list()))):
```

Comment:
You can remove the `()` inside the `any()`, the generator expression works perfectly fine inside it like this: `any(x for x in y)`